test/common/router/retry_state_impl_test.cc

```diff
@@ -505,6 +505,75 @@ TEST_F(RouterRetryStateImplTest, Backoff) {
   EXPECT_EQ(0UL, cluster_.circuit_breakers_stats_.rq_retry_open_.value());
 }
 
+TEST_F(RouterRetryStateImplTest, CustomBackoffInterval) {
```

Comment:
Please comment all new `TEST_F` with single line `//` above explaining what they do (and ideally inside the tests have some running commentary as well). Thanks.